resources/frontend-web/fundamental/js.md

```diff
@@ -0,0 +1,14 @@
+# WELCOME TO JS PART
+<div style="text-align: justify">
+
+Selamat datang di part **`JS`**, jadi kalian disini akan kami beri beberapa referensi untuk mempelajari, memahami, dan mengimplementasikan **`JS`** yang nantikan dipadukan dengan `CSS` dan `HTML`. berikut link tentang HTML:
```

Comment:
```suggestion
Selamat datang di part **`JS`**, jadi kalian disini akan kami beri beberapa referensi untuk mempelajari, memahami, dan mengimplementasikan **`JS`** yang nantikan dipadukan dengan `CSS` dan `HTML`. berikut link tentang JS:
```

There is a typo here i think, its basically just a copy paste mistake

Reply:
whatt wkwk yeah that's right :)

Reply:
master brother suhu fai, I've fixed the typo from `HTML` to `JS` wkwk @fakhrip

Reply:
wkwk nice, thanks dude